tests/utils.py

```diff
@@ -118,3 +122,18 @@ def delete_location(self, driver, app_url, location_name):
         confirm_delete_button = \
             driver.find_element_by_xpath('//input[@type="submit"]')
         confirm_delete_button.click()
+
+
+class TestBase(TestUtilities):
```

Comment:
Why is this not a part of TestUtilites?

Reply:
Oh yeah I just realize that :sweat_smile: